aten/src/ATen/core/Type.h

```diff
@@ -94,7 +94,6 @@ enum class TypeID {
   MSNPULong,
   MSNPUShort,
   MSNPUHalf,
-  MSNPUQInt8,
```

Comment:
why did these go away?  How are backend extensions supposed to implement quantization?